### ISSUE TITLE
plugin seem don't work through package installed

### ISSUE THREAD

**Issue by zhylmzr** [Microsoft/typescript-tslint-plugin#57]
I clone the repository to open it work prefect in VS Code. [capture 1](https://user-images.githubusercontent.com/15730824/52999786-abc81780-3461-11e9-8858-6baceb4315fe.png)
But editor doesn't report everything in a demo. [capture 2](https://user-images.githubusercontent.com/15730824/52999900-fcd80b80-3461-11e9-985a-0de90726eac7.png)

the `tslint.json` copy from the repository and `tsconfig.json` content: 
```json
{
  "compilerOptions": {
    "outDir": "build",
    "module": "es6",
    "target": "es5",
    "sourceMap": true
  }
}
```
some version:
- tslint 5.12.1
- typescript 3.3.1
- vscode-typescript-tslint-plugin 1.0.0

**Comment by mjbvz** [comment]
Can you please share your entire demo repository?

**Comment by zhylmzr** [comment]
@mjbvz there is archive [demo.tar.gz](https://github.com/Microsoft/typescript-tslint-plugin/files/2882607/demo.tar.gz)

**Comment by zhylmzr** [comment]
I found the TSLint must be installed in workspace.
The way to solve my problem.
